test/Microsoft.ML.Tests/Scenarios/HousePricePredictionTests.cs

```diff
@@ -16,16 +16,17 @@ public partial class ScenariosTests : BaseTestClass
          The application will let users submit information about their house, and see a price they could expect if they put the house for sale.
          Because real estate transaction data is public, Contoso has historical data they intend to use to train Machine Learning prediction engine. 
         */
-#pragma warning disable 612
         [Fact(Skip = "Missing data set. See https://github.com/dotnet/machinelearning/issues/3")]
-        public async void PredictHousePriceModelTest()
```

Comment:
In my upcoming PR, I am deleting this test .

This test is being skipped so not not adding to code coverage.  Also, I believe this test was  testing the model  load and prediction path.   

In your PR,  you are using a model  generated using doing .Fit()  but there is already an existing test for it `HousePriceTrainAndPredictionTests.cs`

@codemzs .  Thoughts ?

Reply:
As discussed in person, I am reverting my changes and letting @abgoswam remove all tests that use the housing data set. See PR #2024 for more details.